### PR TITLE
fix(hammerjs): fix window for nodejs

### PR DESCRIPTION
**Pull request by daybrush**
#3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
     "qunit": "phantomjs node_modules/qunit-phantomjs-runner/runner-list.js tests/unit/index.html 15",
     "watch": "run-when-changed --watch 'src/**/*.js' --watch 'tests/unit/*.js' --watch 'tests/unit/gestures/*.js' --exec 'npm run watch-tasks'",
     "watch-tasks": "npm run rollup && npm run string-replace && npm run uglify && npm run jshint && npm run jscs",
-    "test": "npm run build && npm run qunit"
+    "test": "npm run build && npm run qunit",
+    "test:node": "node ./dist/hammer.js"
```
Comment (jongmoon) [inline package.json:64]: @younkue how about to comment what is this test for user convinience.

File: src/utils/utils-consts.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
+
 const VENDOR_PREFIXES = ['', 'webkit', 'Moz', 'MS', 'ms', 'o'];
-const TEST_ELEMENT = document.createElement('div');
+const TEST_ELEMENT = typeof document === "undefined" ? {style: {}} : document.createElement('div')
```
Comment (jongmoon) [inline src/utils/utils-consts.js]: @younkue semicolon is omitted.
and why style is defined where document type is undefiend.

Comment (daybrush) [reply src/utils/utils-consts.js]: use style of TEST_ELEMENT in get-touchaction-props.js

**Review by jongmoon — APPROVED** [review]
LGTM

**Pull request merged**